### DISCUSSION TITLE
Doing world backups without shutting down server.

### DISCUSSION THREAD

**Discussion by turbo-david** [Nimdy/Dedicated_Valheim_Server_Script#218] [Q&A]
I recently moved to the server script here. It seems like backup only works by first shutting down the server. (my old server i would just copy the files to a local nas) I'm assuming shutting down the server is to protect the server from writing the file as your copying it? I was doing 30 min backups but I can't be restarting the server that often in the current setup.

**Comment by Nimdy** [comment]
If your file system is writing correctly and your .db and .fwl are writing correctly, you should only need to do backups once a day.

Backups are up to you, we are just providing a manual simple way.

If you want to make your own autobackup:

https://github.com/Nimdy/Dedicated_Valheim_Server_Script/wiki/How-to-set-automatic-backups-for-Valheim-Server-World-Data-in-Ubuntu
